specification/sbi_atee.adoc

```diff
@@ -227,14 +1486,6 @@ page is not-present in the second stage page table and not tracked as a TEE
 page. The VMM owns the process of reclaiming the 
 page.                            
 
-| teecall_tg_page_accept       | Intrinsic used by the TVM to accept pages 
-as part of initial (pre-init) load or post-init add/conversion of memory 
-from non-confidential to confidential.The page mapping stays in a pending 
-state (where TLB entries cannot be created) until this operation succeeds. 
-The TSM enforces the page is zero-cleared during the page_post_add 
-performed by the VMM when the page is (re)added back as a confidential TVM 
-page.  
-
```

Comment:
we should add this back as part of the dynamic shared memory conversion support.

Reply:
What was the API (sorry, can't see the deleted change for some reason)?

Reply:
(this is tg_accept, FYI)

We should continue the discussion of shared conversion on the doc; I'm not sure we need to introduce guest memory acceptance just for the purposes of memory un-sharing.

Reply:
@abrestic-rivos was this updated here? do we need to revert the deleted lines above?

Reply:
resolved - we used the guest-side blocking TLB flush approach instead of accept.